platform-sdk/docs/base/logging/logging.md

```diff
@@ -0,0 +1,177 @@
+
+To provide developers with more control and flexibility in logging, the configuration supports filters for markers. This feature allows developers to focus on log messages associated with specific markers, regardless of the logger's log level. Here's a sample configuration for marker filters:
+
+```properties
+# Marker filter configuration
+logging.marker.CONFIG = ENABLED
+logging.marker.CRYPTO = DISABLED
+logging.marker.OTHER = DEFAULT
+```
+
+### Markers
+
+In this configuration, the `logging.marker.NAME` pattern is used, where `NAME` represents the marker's name, and the value can be set to `ENABLED`, `DISABLED`, or `DEFAULT`. For example, `logging.marker.CONFIG = ENABLED` would ensure that all log messages tagged with the `CONFIG` marker are displayed, irrespective of their log level.
+
+A typical logging configuration file, incorporating these marker filters, might look like this:
+
+```properties
+# General logging level configuration
+logging.level = INFO
+logging.level.com.swirlds.common.crypto = DEBUG
+logging.level.com.swirlds.common.crypto.Signature = WARN
+logging.level.com.hashgraph = WARN
+
+# Marker-specific logging configuration
+logging.marker.CONFIG = ENABLED
+```
+
+### Handlers
+
+To further refine our logging system, we have incorporated handlers for more granular control over logging behavior. Each handler can be distinctly named and configured using the prefix `logging.handler.NAME`, where `NAME` serves as a unique identifier. This structure allows for the application of handler-specific settings. For instance, `logging.handler.NAME.level` is used to set the logging level for a specific handler, ensuring that all previously discussed features such as marker filters and log level settings are compatible. This setup is instrumental in creating dedicated log files or outputs for specific types of log messages, offering a focused view that is particularly useful in complex systems or during targeted analyses. 
+
+A key aspect of the Logging System design is its performance, emphasizing maximum efficiency with as low an impact as possible on the running system.
+
+A particularly useful feature of handlers is the `inheritLevels` property. This boolean property determines whether the handler should inherit the default level configurations set globally. By default, `inheritLevels` is set to `true`. However, it can be turned off to create a handler that focuses exclusively on a specific aspect of logging, such as a particular marker. For example, if you need a handler that only logs entries marked with the `CRYPTO` marker, your configuration would look like this:
```

Comment:
Can you do one sentence per line? Otherwise the review will become very complexe since a comment can only be done for several sentences.